daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

```diff
@@ -135,7 +135,7 @@ Because workflow retry policies are configured in code, the exact developer expe
 | --- | --- |
 | **Maximum number of attempts** | The maximum number of times to execute the activity or child workflow. |
 | **First retry interval** | The amount of time to wait before the first retry. |
```

Comment:
Given that we are in this doc, can you add the time units for all the rows in this table. For example is this in seconds, minutes etc.

Reply:
Conceptually doesnt make sense to document here the time units for this as it will depend on the SDK implementation, each language has its own way of expressing time durations, i.e in golang its configured with `10*time.Millisecond` or `1*time.Hour`